data/campaigns/fri03.wmf/scripting/tribes/diker/init.lua

```diff
@@ -0,0 +1,54 @@
+dirname = "campaigns/fri03.wmf/" .. path.dirname (__file__)
+
+push_textdomain("scenario_fri03.wmf")
+
+descriptions:new_worker_type {
+   name = "frisians_diker",
+   -- TRANSLATORS: This is a worker name used in lists of workers
+   descname = pgettext ("frisians_worker", "Diker"),
+   icon = dirname .. "menu.png",
+   vision_range = 2,
+
+   programs = {
```

Comment:
we should have a find_suitable program as the diker wastes log even if there is no suitable diking spot left. Should be the same like for amazon gardening center

Reply:
fixed